app/src/componentLibrary/modal/modalLayout/modalLayout.jsx

```diff
@@ -113,6 +117,7 @@ ModalLayout.propTypes = {
   modalSize: PropTypes.oneOf(['default', 'small', 'large']),
   onClose: PropTypes.func,
   overlay: PropTypes.string,
+  isFieldsChanged: PropTypes.bool,
```

Comment:
It is not clear what this property will lead to.
I suggest to rename it according to its purpose i.e. `allowCloseOutside` or something like that.